frontend/views/containers/proposals/ProposalsWidget.vue

```diff
@@ -170,6 +183,11 @@ export default ({
 
       this.openModal(modalNameMap[itemId], queries[itemId] || undefined)
     }
+  },
+  watch: {
+    currentGroupId () {
+      this.updateArchivedProposals()
```

Comment:
Nice fix here that makes sure the different groups don't share archived proposals while switching the groups around.